src/scripts/exclusion_term_expansion.py

```diff
@@ -297,13 +297,19 @@ def run(
         df_results = pd.DataFrame()
         df_results['term_id'] = ''
 
+    # Filter out non-owned
```

Comment:
@matentzn I just now filtered results so any non-owned terms don't show up in the output. `NCBITaxon:9606`(not sure how this one got there because it's not in `config/omim_exclusions.tsv`) and `SO:0000704` (gene) were in the output. I figure that these outputs shouldn't include terms foreign to the ontology, but if that's incorrect, let me know.

Reply:
I am not sure if this is _always_ correct, but for all ontologies we are dealing with so far, you are right.